<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CD Audio Plugin Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="deviceCheckBox">
     <property name="text">
      <string>Override device:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="deviceLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="speedCheckBox">
     <property name="text">
      <string>Limit cd speed:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="speedSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>48</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cdtextCheckBox">
     <property name="text">
      <string>Use cd-text</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deviceCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>deviceLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>speedSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
